### DISCUSSION TITLE
Three pane view is unavailable when using APIDocMdx

### DISCUSSION THREAD

**Discussion by malathikrishnap** [rohit-gohri/redocusaurus#332] [Q&A]
Hi there,

I'm importing a spec using ApiDocMdx. This displays only two panes and not three. I noticed the same in the examples you stated in https://redocusaurus.vercel.app/docs/nested/nested-1#tag/pet/operation/newPet. Is there a way to display all the three panes with the existing sidebar?

`import ApiDocMdx from '@theme/ApiDocMdx';
<ApiDocMdx id="create-connector-group" />`

However, if I load the spec using "route", for example, `localhost:3000/api/createconnectors`, I can see all the three panes.

**Comment by rohit-gohri** [comment]
Could you share a screenshot of what you are getting and what is expected? I don't really understand the problem

**Reply by malathikrishnap** [reply]
Apologies!
Here're the screenshots:

<img width="1303" alt="ThreePaneView" src="https://github.com/rohit-gohri/redocusaurus/assets/112460089/a71e2c64-b257-4a26-8247-9d3629062fd2">
<img width="1063" alt="TwoPaneView" src="https://github.com/rohit-gohri/redocusaurus/assets/112460089/090d4c3e-1a1e-45cb-acc0-7d2b80fcee4d">

**Reply by rohit-gohri** [reply]
Ohh.. Got it. You can change the breakpoints in the redoc theme settings - https://github.com/Redocly/redoc/blob/main/docs/config.md#theme-settings . The breakpoints decide when to switch b/w 2 or 3 panes. 

The settings are applied using `redocly.yaml` file, see https://redocusaurus.vercel.app/docs/getting-started/Installation#config-optional
